packages/block-library/src/table/edit.js

```diff
@@ -417,3 +469,10 @@ export default class TableEdit extends Component {
 		);
 	}
 }
+
+const ComposedTableEdit = compose( [
```

Comment:
In other block files I've seen the composed version be defined on the same line it is exported (so it is not given a name). Just noting for the sake of consistency.